apps/dashboard/components/agents/agent-details.tsx

```
@@ -22,6 +33,32 @@ export function AgentDetails({
     model: agent.model || availableModels[0].id,
   });
   const descriptionSize = countBytes(formData.description || '');
+  const calculateResourceSize = (resource: TypedResource<ResourceType>) => {
+    switch (resource.type) {
+      case 'contract':
+        const value = resource.value as ContractValue;
```

Biome (1.9.4)

[error] Other switch clauses can erroneously access this declaration.
Wrap the declaration in a block to restrict its access to the switch clause.

The declaration is defined in this switch clause:

Unsafe fix: Wrap the declaration in a block.


(lint/correctness/noSwitchDeclarations)